hazelcast/src/main/java/com/hazelcast/client/ClientEngine.java

```diff
@@ -59,6 +60,13 @@
     SecurityContext getSecurityContext();
 
     /**
+     * Returns the SerializationService.
+     *
+     * @return the SerializationService
+     */
+    SerializationService getSerializationService();
```

Comment:
is RingBuffer the only data structure which needs to expose `SerializationService`?